pkg/controller/node/node_controller.go

```diff
@@ -187,24 +220,73 @@ func (r *ReconcileNode) createNsxClients() (*NsxClients, error) {
 	var managerHost = strings.Split(cfg.Section("nsx_v3").Key("nsx_api_managers").Value(), ",")[0]
 	var user = cfg.Section("nsx_v3").Key("nsx_api_user").Value()
 	var password = cfg.Section("nsx_v3").Key("nsx_api_password").Value()
-
+	var insecure = false
+	if strings.ToLower(cfg.Section("nsx_v3").Key("insecure").Value()) == "true" {
+		insecure = true
+	}
 	nsxClients := NsxClients{}
 	nsxClients.Cluster = cluster
 
+	// Get cert/key/ca, then write then into temp files
+	var certData = cfg.Section("operator").Key("nsx_api_cert").Value()
+	var keyData = cfg.Section("operator").Key("nsx_api_private_key").Value()
+	var caData = cfg.Section("operator").Key("nsx_ca").Value()
+	certDataDecoded, err := base64.StdEncoding.DecodeString(certData)
```

Comment:
nit: 234-245, 255-266 can make use of a func

Reply:
Done